search/lib/util/app.js

```diff
@@ -11,9 +11,27 @@ const extractParam = (queryStringParams, param, defaultVal = null) => {
   return defaultVal;
 };
 
-const createLink = (rel, href, title, type = 'application/json') => ({
-  href, rel, type, title
-});
+// function createLink(rel, href, title, type = 'application/json') {
+
+// }
```

Comment:
found another commented out line to remove.

Reply:
Deleted.